cookbook/core/basic/basic_workflow.py

```diff
@@ -12,7 +12,7 @@
 of ``t2`` in the workflow below. As such, the body of workflows is run at "registration" time. Please refer to the
 registration docs for additional information as well since it is actually a two-step process.
 
-Take a look at the conceptual `discussion <https://lyft.github.io/flyte/user/concepts/workflows_nodes.html#workflows>`__
+Take a look at the conceptual `discussion <https://docs.flyte.org/en/latest/dive_deep/workflows_nodes.html>`__
```

Comment:
Can't we use cross referencing here to avoid linking to the page?
@cosmicBboy @kumare3